src/rgw/rgw_common.h

```diff
@@ -547,14 +555,16 @@ struct RGWUserInfo
   RGWQuotaInfo bucket_quota;
   map<int, string> temp_url_keys;
   RGWQuotaInfo user_quota;
+  uint32_t type;
```

Comment:
I have left this as uint32_t since:
1. decode doesn't recognize enums
2. I followed the code for rgw_meta_sync_info and there also the type is uint.